frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/create-sc.tsx

```diff
@@ -316,12 +320,20 @@ const CreateSC: React.FC<CreateSCProps> = ({ match, hasNoProvSC, mode, lsoNs })
             className="co-alert ocs-install-info-alert"
             variant="info"
             isInline
-            title={!hasNoProvSC ? 'Missing storage class' : 'Internal - Attached devices'}
+            title={
+              !hasNoProvSC
+                ? t('ceph-storage-plugin~Missing storage class')
+                : 'Internal - Attached devices'
```

Comment:
missing `t`

Reply:
We are suggested by marketing team to not i18n this the ocs modes  - Internal/Internal-Attached devices/External.
Hence keeping it in plain english